src/opnsense/mvc/app/models/OPNsense/Kea/KeaDhcpv4.xml

```diff
@@ -28,6 +28,15 @@
                     <raw>raw</raw>
                 </OptionValues>
             </dhcp_socket_type>
+            <kaffinity type="BooleanField">
+                <Default>0</Default>
+                <Required>Y</Required>
+            </kaffinity>
+            <hold_reclaimed_time type="IntegerField">
+                <default>28800</default>
+                <Required>Y</Required>
+                <MinimumValue>0</MinimumValue>
+            </hold_reclaimed_time>
```

Comment:
The lightweight approach would be just to offer hold_reclaimed_time, make it optional to retain the current default and let the default values be set implicitly. If defaults change on Keas end for good reasons we will not be able to change them, especially hard for values that are not presented on the GUI then.